chester/catalog-applications/chester-range.md

```diff
@@ -0,0 +1,241 @@
+---
+slug: chester-range
+title: CHESTER Range
+---
+import Image from '@theme/IdealImage';
+
+# CHESTER Range
+
+This article describes the core functionality, hardware description, and
+example **JSON** message of the catalog application **CHESTER Range**.
+
+:::caution
+
+Some of the basics are not provided, as they are common for all **CHESTER**
+catalog applications. For example, see the article [**Platform
+Management**](../category/platform-management) on how to work with the
+interactive console.
+
+:::
+
+## Application Overview
+
+**CHESTER Range** measures distance using the [MaxBotix
+MB7066](https://shop.hardwario.com/ultrasonic-sensor/) ultrasonic sensor. It
+can also measure temperature using the DS18B20 one-wire sensor and humidity
+using CHESTER S2.
+
+## Application Variants
+
+**CHESTER Range** can be ordered in one of these variants:
+
+### CHESTER Range
+
+The catalog **CHESTER Range** hardware consists of the following ordering
+codes:
+
+* `CHESTER-M-CGLS` - Standard mainboard
+
+* `CHESTER-X0A:A` - Input module (4 channels)
+
+See [**Ordering Codes**](../ordering-codes.md) for more details.
+
+Firmware build shield options: `ctr_lte ctr_mb7066_a`
+
+### CHESTER Range Z
+
+The catalog **CHESTER Range Z** hardware consists of the following ordering
+codes:
+
+* `CHESTER-M-CGLS` - Standard mainboard
+
+* `CHESTER-X0A:A` - Input module (4 channels)
+
+* `CHESTER-Z` - Backup module
+
+See [**Ordering Codes**](../ordering-codes.md) for more details.
+
+Firmware build shield options: `ctr_lte ctr_mb7066_a ctr_z`
+
+## Measurement and Behavior
+
+- All sensors are **sampled** with a configurable period (parameter
+  `interval-sample`).
+- Samples are then **aggregated** in the configurable interval. Minimum,
```

Comment:
Do not break lines manually; please keep the whole paragraphs on the same line, and use word-wrapping in the editor. Otherwise, it is a never-ending mess with formatting when maintaining the article.